oozie/src/main/java/org/apache/falcon/oozie/feed/FeedReplicationWorkflowBuilder.java

```diff
@@ -145,6 +161,24 @@ private String getDefaultMapBandwidth() {
         return RuntimeProperties.get().getProperty("falcon.replication.workflow.mapbandwidth", "100");
     }
 
+    private String getDefaultAmMemory() {
```

Comment:
Its just not ProcessWorkflowBuilder , there are data import, export and hive builders.